src/pgzero/game.py

```diff
@@ -256,6 +256,9 @@ def key_down(event):
             if event.key == pygame.K_q and \
                     event.mod & (pygame.KMOD_CTRL | pygame.KMOD_META):
                 sys.exit(0)
+            # Default key for screenshots is F12.
+            if event.key == pygame.K_F12:
+                pgzero.screen.screen_instance.screenshot()
```

Comment:
If there were errors writing this screenshot we should catch the exception and print a traceback.

But only here - if the user calls `screen.screenshot()` they should receive the exception.